extensions/jaeger/deployment/src/main/java/io/quarkus/jaeger/deployment/JaegerProcessor.java

```diff
@@ -43,4 +50,51 @@ public void build(BuildProducer<FeatureBuildItem> feature) {
         feature.produce(new FeatureBuildItem(FeatureBuildItem.JAEGER));
     }
 
+    private void produceMetrics(BuildProducer<MetricBuildItem> producer) {
```

Comment:
Is there any test that checks that these metrics are actually reported by Jaeger and alternatively also report that new metrics are reported by Jaeger client?

Reply:
None that I'm aware of, Jaeger tests in Quarkus codebase are extremely basic because they don't spin up an actual Jaeger instance so there's not much that we can actually test... That would need a wholly new test suite I think.

Reply:
We could probably use `testcontainers` for that - we do use it in some places.

Reply:
I thought that these metrics were reported by the Jaeger client not the server.

This article might help to test jaeger with test containers https://medium.com/jaegertracing/generating-jaeger-grpc-services-and-using-jaeger-in-junit-with-testcontainers-ac08a6e4e1c

Reply:
The metrics are computed by the client, but they seem to be updated after succesfully reporting them to the server, so if there's no server connection, then the generated metrics don't make much sense.

Reply:
I reported https://github.com/quarkusio/quarkus/issues/8680 for the new tests so we don't forget about it